### PR TITLE
add keepalive settings to grpc server

### PR DESCRIPTION
**Pull request by maksim-paskal**
Add keepalive settings to grpc server to reduce reconnection Envoy to control-plane

#449

### CODE REVIEW

File: docs/README.md
Comment (alecholmez) [inline docs/README.md:39]: I think you can remove this since we set it in the below `append()`

File: internal/example/server.go
Comment (alecholmez) [inline internal/example/server.go:61]: this can be removed

File: pkg/test/server.go
Comment (alecholmez) [inline pkg/test/server.go:65]: This can be removed

**Review by alecholmez — CHANGES_REQUESTED** [review]
Thanks for this PR, what's the reasoning behind the times picked for each timeout and keepalive setting? 

Just a few minor nits to remove some duplication

**Comment by maksim-paskal** [comment]
@alecholmez Thanks for review, I got this times from Envoy [documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/examples#dynamic) for dynamic bootstrap configuration

File: pkg/test/server.go
Comment (alecholmez) [inline pkg/test/server.go:63]: Sorry one more thing, under this comment could you maybe add a note regarding the envoy docs mentioning the keepalive settings? Same with `internal/example` code as well. This should be good to go after this

**Review by alecholmez — CHANGES_REQUESTED** [review]
This looks good thank you!

File: pkg/test/server.go
Comment (maksim-paskal) [reply pkg/test/server.go]: Looks good, please review my changes.

**Review by alecholmez — APPROVED** [review]
Great thank you! Once the build runs I'll merge

**Comment by maksim-paskal** [comment]
I change my `Signed-off-by` commit message - please make review one more time

**Review by alecholmez — APPROVED** [review]

**Pull request merged**
